### ISSUE TITLE
ParticleAttractor2D causes errors when exiting tree

### ISSUE THREAD

**Issue by Keetz** [godotengine/godot#9503]
**Operating system or device - Godot version:**
Godot 2.1.3-stable

**Issue description:**
When a ParticleAttractor2D and its Particle2D exits the tree no 'exit_tree' signal exists and it fails calling ParticleAttractor2D::_owner_exited

```
ERROR: emit_signal: Error calling method from signal 'exit_tree': 'ParticleAttractor2D::_owner_exited': Method not found.
   At: core/object.cpp:1162.

ERROR: disconnect: Nonexistent signal: exit_tree
   At: core/object.cpp:1415.
```

**Steps to reproduce:**
1. Create a new scene
2. Add a Particle2D and a ParticleAttractor2D
3. Assign the Particle2D to the 'Particle Path' on the ParticleAttractor2D
4. Save and exit the scene

**Comment by akien-mga** [comment]
Relevant code: https://github.com/godotengine/godot/blob/b46d7f9/scene/2d/particles_2d.cpp#L94-L111

I think it might just lack a binding of `_owner_exited` in `_bind_methods`,

**Comment by akien-mga** [comment]
Likely does not affect 3.0 as Particles2D was rewritten and the relevant code no longer exists (no ParticleAttractor2D that I can see).

**Comment by akien-mga** [comment]
Fixed by #9505.
